### PR TITLE
84 deleteupdate recipes

### PR DESCRIPTION
**Pull request by MalikHenry1**
Worked on with Jamie and Rashaad. Delete/Update now functional,

### CODE REVIEW

File: public/js/drink.js [hunk 1]
```diff
@@ -89,9 +89,36 @@ document
...
+    }),
+    headers: {"Content-Type": "application/json"}
+  });
+
+  document.location.reload();
+}
 
-
-  
+$('.fa').on('click', starRatingHandler);
```
Comment (heatherknoyes) [inline public/js/drink.js:121]: What class uses this in the front end? We would want this to be associated with a review on the front end so I'm a little confused at what element would have class .fa.

File: views/recipe.handlebars [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<div class="recipe-form">
```
Comment (heatherknoyes) [inline views/recipe.handlebars:1]: This whole page does not exist in the most recent iteration so I would remove this from the PR.

**Review by heatherknoyes — CHANGES_REQUESTED** [review]

File: public/js/drink.js (hunk 1 above)
Comment (MalikHenry1) [reply public/js/drink.js:121]: The .fa class is for each star imported from font-awesome.

File: views/recipe.handlebars (hunk 2 above)
Comment (MalikHenry1) [reply views/recipe.handlebars:1]: Got it.

**Pull request closed without merging**